app/features/analyst/ciip-filings.feature

```diff
@@ -0,0 +1,16 @@
+@mvp
+Feature: Authentication
+
+  As an incentive analyst
+  I want triage the received CIIP filings list
+  So that I can analyze filed CIIP applications
+
+  Scenario: view dashboard of all received CIIP filings
+  Scenario: filter dashboard of all received CIIP filings by year
+  Scenario: filter dashboard of all received CIIP filings by status
+  Scenario: view details of a received CIIP filing
```

Comment:
Does viewing the details of a received CIIP filing include exposing the changes made by the industry reporter to SWRS imported data?  If so, should we be more explicit?  If not, can we add or is this not what we want anymore?

Reply:
fixed

Reply:
Is that diff of SWRS<=>CIIP data in scope for our MVP though? @Maralsotoudehnia @hamzajaved

Reply:
We clarified this scope in our MVP discussion and wrote it on the paper as "expose changes to SWRS made in CIIP application (for analyst, not compliance)"